.size-limit.js

```diff
@@ -7,7 +7,7 @@ module.exports = [
     name: 'Core - CDN',
     path: 'dist/legacy/rudder-analytics.min.js',
     gzip: true,
-    limit: '37.1 kB',
+    limit: '41 kB',
```

Comment:
There is no change in the core SDK still, why did you change these values?

Reply:
I have not updated the packages so it was getting increased. Fixed that.